libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -710,17 +710,17 @@ string IRGenerator::initStateVariables(ContractDefinition const& _contract)
 }
 
 
-void IRGenerator::generateImplicitConstructors(ContractDefinition const& _contract)
+void IRGenerator::generateConstructors(ContractDefinition const& _contract)
 {
 	auto listAllParams = [&](
 		map<ContractDefinition const*, vector<string>> const& baseParams) -> vector<string>
-	{
-		vector<string> params;
-		for (ContractDefinition const* contract: _contract.annotation().linearizedBaseContracts)
-			if (baseParams.count(contract))
-				params += baseParams.at(contract);
-		return params;
-	};
+		{
+			vector<string> params;
+			for (ContractDefinition const* contract: _contract.annotation().linearizedBaseContracts)
+				if (baseParams.count(contract))
+					params += baseParams.at(contract);
+			return params;
+		};
```

Comment:
Why this change?

Reply:
Really? I thought it should line up with the top line

Reply:
It should but the top line is wrapped in an unusual way. It would normally be wrapped like this:
```c++
	auto listAllParams = [&](
		map<ContractDefinition const*, vector<string>> const& baseParams
	) -> vector<string>
	{
		...
	}
```